From d9ecd27f41bf9d7e97eef213f815c18877f95742 Mon Sep 17 00:00:00 2001
From: Luca Abeni <lucabe72@gmail.com>
Date: Sat, 19 Nov 2016 23:42:28 +0100
Subject: [PATCH 7/7] Fix WARNS

---
 kernel/sched/deadline.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/kernel/sched/deadline.c b/kernel/sched/deadline.c
index be08895..29e586f 100644
--- a/kernel/sched/deadline.c
+++ b/kernel/sched/deadline.c
@@ -50,7 +50,7 @@ void add_running_bw(struct sched_dl_entity *dl_se, struct dl_rq *dl_rq)
 
 	lockdep_assert_held(&(rq_of_dl_rq(dl_rq))->lock);
 	dl_rq->running_bw += dl_se->dl_bw;
-	SCHED_WARN_ON(dl_rq->running_bw < old); /* overflow */
+	WARN_ON(dl_rq->running_bw < old); /* overflow */
 }
 
 static inline
@@ -60,7 +60,7 @@ void sub_running_bw(struct sched_dl_entity *dl_se, struct dl_rq *dl_rq)
 
 	lockdep_assert_held(&(rq_of_dl_rq(dl_rq))->lock);
 	dl_rq->running_bw -= dl_se->dl_bw;
-	SCHED_WARN_ON(dl_rq->running_bw > old); /* underflow */
+	WARN_ON(dl_rq->running_bw > old); /* underflow */
 }
 
 static void task_go_inactive(struct task_struct *p)
-- 
2.7.4

